lib/ansible/modules/windows/win_route.py

```diff
@@ -35,11 +35,11 @@
 options:
   destination:
     description:
-      - Destination IP address in CIDR format (ip address/prefix lenght)
+      - Destination IP address in CIDR format (ip address/prefix length)
     required: true
   gateway:
     description:
-        - Gateway used by the static route. If gateway is not provided it will be set to "0.0.0.0"
+        - Gateway used by the static route. If the gateway is not provided it will be set to "0.0.0.0"
```

Comment:
This should actually become:
```yaml
    - The gateway used by the static route.
    - If C(gateway) is not provided it will be set to C(0.0.0.0).
```